pipeline/05-finalize.R

```diff
@@ -203,7 +205,23 @@ tictoc::tic.clearlog()
 
 
 #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-# 4. Upload --------------------------------------------------------------------
+# 4. Generate performance report -----------------------------------------------
+#- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+message("Generating performance report")
+
+here("reports/performance/performance.qmd") %>%
+  quarto_render(
+    execute_params = list(
+      run_id = run_id,
+      year = params$assessment$year
+    )
+  )
```

Comment:
**thought:** One big issue I foresee with this is that any failure in the Quarto doc is also going to kill the entire pipeline. I foresee a model run that's been running for 50 hours being killed by a misplaced comma in the doc :(

**suggestion (blocking):** I recommend that we add some kind of error handling here to prevent the doc from killing the pipeline. The result of the error handling should be included in the SNS notification.